controllers/azurejson_machinepool_controller.go

```diff
@@ -114,7 +114,6 @@ func (r *AzureJSONMachinePoolReconciler) Reconcile(ctx context.Context, req ctrl
 		}
 		return reconcile.Result{}, err
 	}
-
```

Comment:
nit: You probably didn't mean to make this change

Reply:
Updated this to include the space unintended for this change.  Resolving.